### ISSUE TITLE
Adding ids to csv files

### ISSUE THREAD

**Issue by mmc46** [VirtualFlyBrain/VFB#921]
It would be useful if the download files included ids. 

For example, in a transgene expresison query, the current columns are

Driver/reporter	| Expressed in	| Reference	| Source|
--------------------|-------------------|--------------------|-----------|
P{GMR59D04-GAL4}	|adult pars intercerebralis	|Williams et al., 2014, Europ. J. Neurosci. 40(3): 2513--2526	|FlyLight_Brain

I suggest adding the id columns for driver/reporter and reference (PMID ? doi ?) and maybe anatomy.

Driver/reporter	| Driver/reporter id| Expressed in	| Anatomy id (?)| Reference	| Reference id| Source|
--------------------|------------|-------|-----------|---------|-----------|------| 
P{GMR59D04-GAL4}| FBtp0061640 |adult pars intercerebralis	| FBbt:00003700 (?) | Williams et al., 2014, Europ. J. Neurosci. 40(3): 2513--2526	| 10.1111/ejn.12619| FlyLight_Brain

**Comment by dosumis** [comment]
Anatomy ID should be available already.  PMID/FBrf etc can be pulled from neo4J.

**Comment by Robbie1977** [comment]
Agreed, might delay a few weeks as got to look at revamping reports via neo4j completely iminantly. Good opertunity to look at all queries and consider what queries we'd ideally like and what they should contain. If you get two seconds tomorrow you two might want to discuss potential for new queries with neo4j?
